Mathlib/MeasureTheory/Group/FundamentalDomain.lean

```diff
@@ -719,4 +727,281 @@ protected theorem fundamentalInterior : IsFundamentalDomain G (fundamentalInteri
 
 end IsFundamentalDomain
 
+section FundamentalDomainMeasure
+
+variable (G) [Group G] [MulAction G α] [MeasurableSpace α]
+  (μ : Measure α)
+
+local notation "α_mod_G" => MulAction.orbitRel G α
+
+local notation "π" => @Quotient.mk _ α_mod_G
+
+variable {G}
+
+@[to_additive addQuotientMeasure_apply]
+lemma quotientMeasure_apply (s : Set α)
```

Comment:
It would help if you explained somewhere (in the docstring for the lemma and/or the file-level docstring) what you mean by a quotient measure (i.e., restricting to a set and then pushing foward), because the lemma name is impossible to guess without this piece of information.

Reply:
Yes, we don't need this lemma after all. Removing it.